### PR TITLE
Changed misleading name of DisposeSynchronizedScene

### PR DESCRIPTION
**Pull request by freedy69**
Used real name of this native because DisposeSynchronizedScene is misleading and not at all what this native does

### CODE REVIEW

**Comment by PsychoShock** [comment]
You could add a description if you want too. `Set the duration of a synchronized scene to the script that invokes this native.`

File: PED/TakeOwnershipOfSynchronizedScene.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+## TAKE_OWNERSHIP_OF_SYNCHRONIZED_SCENE
+
+```c
+// 0xCD9CC7E200A52A6F 0xBF7F9035
+void TAKE_OWNERSHIP_OF_SYNCHRONIZED_SCENE(int scene);
+```
+
+
+## Parameters
+* **scene**: Scene ID returned by (CREATE_SYNCHRONIZED_SCENE)(#_0x8C18E0F9080ADD73)
```
Comment (4mmonium) [inline PED/TakeOwnershipOfSynchronizedScene.md:14]: Hey, this is wrong, it's supposed to be `[CREATE_SYNCHRONIZED_SCENE](#_0x8C18E0F9080ADD73)`

Thought you should know. 😀

**Review by 4mmonium — CHANGES_REQUESTED** [review]
Hey there, requested some changes. Thank you! 😊

File: PED/TakeOwnershipOfSynchronizedScene.md (hunk 1 above)
Comment (freedy69) [reply PED/TakeOwnershipOfSynchronizedScene.md]: Changed it now, is it okay?

File: PED/TakeOwnershipOfSynchronizedScene.md [hunk 2]
```diff
@@ -11,5 +11,5 @@ void TAKE_OWNERSHIP_OF_SYNCHRONIZED_SCENE(int scene);
 
 
 ## Parameters
-* **scene**: Scene ID returned by (CREATE_SYNCHRONIZED_SCENE)(#_0x8C18E0F9080ADD73)
+* **scene**: Scene ID returned by [`CREATE_SYNCHRONIZED_SCENE`](#_0x8C18E0F9080ADD73)
```
Comment (PsychoShock) [inline PED/TakeOwnershipOfSynchronizedScene.md:14]: Remove `

Comment (freedy69) [reply PED/TakeOwnershipOfSynchronizedScene.md:14]: i mean i've seen a lot of other natives use this

Comment (4mmonium) [reply PED/TakeOwnershipOfSynchronizedScene.md:14]: > Remove `

Unneeded, both are correct 😅

**Review by 4mmonium — APPROVED** [review]
LGTM, thanks!

**Pull request merged**